migrations/900-remove-in-app.sql

```diff
@@ -0,0 +1,2 @@
+/* These will be recreated again automagically with the new data. */
+delete from inapp_products where simulate is not null;
```

Comment:
Ah that works on prod where there's only two values with simulate in. But i'll change thats cool.